core/src/main/java/ai/timefold/solver/core/config/solver/SolverConfig.java

```diff
@@ -210,6 +212,7 @@ public class SolverConfig extends AbstractConfig<SolverConfig> {
     // Warning: all fields are null (and not defaulted) because they can be inherited
     // and also because the input config file should match the output config file
 
+    protected List<PreviewFeature> enablePreviewFeatureList = null;
```

Comment:
Shouldn't this be an `EnumSet`?

Reply:
We don't use `EnumSet` in any other configuration class. I believe we will need an adapter for it.